pkg/provisioning/config/yaml/linter.go

```diff
@@ -21,23 +21,31 @@ import (
 
 	"github.com/Masterminds/semver/v3"
 	"github.com/conduitio/conduit/pkg/foundation/log"
+	"github.com/conduitio/conduit/pkg/provisioning/config/yaml/internal"
 	"github.com/conduitio/yaml/v3"
 )
 
 type configLinter struct {
-	version  string
-	warnings warnings
+	changelog internal.Changelog
+
+	expandedChangelog map[string]map[string]any
+	version           string
```

Comment:
`expandedChangelog` contains all versions, and `version` is just the version being "linted"?

Reply:
Yes! I'll add a comment 😅

Reply:
I added a comment in the last PR to prevent more rebases. `version` was actually removed in the next PR because the linter gets the version from the parser, I added a comment for `expandedChangelog` though (https://github.com/ConduitIO/conduit/pull/929/commits/1dfbabeabfc1fd8f2915fc1b87afe56188296eb4).